app/plot_app/configured_plots.py

```diff
@@ -195,12 +195,11 @@ def generate_plots(ulog, px4_ulog, db_data, vehicle_data, link_to_3d_page,
     data_plot.change_dataset('vehicle_global_position')
     data_plot.add_graph(['alt'], colors8[2:3], ['Fused Altitude Estimation'])
     data_plot.change_dataset('position_setpoint_triplet')
-    data_plot.add_circle(['current.alt'], [plot_config['mission_setpoint_color']],
-                         ['Altitude Setpoint'])
-    data_plot.change_dataset(actuator_controls_0.thrust_sp_topic)
-    if actuator_controls_0.thrust_z_neg is not None:
```

Comment:
Can you separate the commit and provide the reason for the change for future reference?
We could also shift the thrust into the range of the altitude instead.

Reply:
Split the commit and added a comment.

Reply:
https://github.com/PX4/flight_review/pull/287